commands/dev-deploy.js

```diff
@@ -36,7 +36,7 @@ module.exports = {
 };
 
 async function devDeploy(options) {
-    await eventtracking.askForConsentIfNeeded(options);
```

Comment:
I'd prefer if we keep `askForConsentIfNeeded` name as it is open ended and allows to add more questions if needed. We don't want to rename it to `askForOptInAndAccountIDAndGeoLocation` next time we want to collect some other data.